aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java

```diff
@@ -122,6 +122,14 @@ private DescribePatchBaselinesResponse describePatchBaselinesResponse(final Stri
                 .stream().map(baseline -> baseline.baselineId()).collect(Collectors.toList());
 
         for (String baselineId : baselineIdList) {
+
+            if(baselineId.contains("/"))
+            {
```

Comment:
why would baselineId contains "/", what is the use case?

Reply:
??

Reply:
The baselineid should be alpha numeric value. 
When i do the contract test, For the baselineId it is passing the patchbasline endpoint url instead of patch baseline id. In that case the process is getting failed. So we have made the changes according to request formate.

1) If the baseline id is default , aplha numeric word then the process will still run the same process
2) If the baselineId is getting in the formate of endpoint URL type and we will parse the data based on the delimiter.

For the above senario's i have made the changes and if any concerns please let me know.

Reply:
Can you please suggest me necessary changes to be made there?

Reply:
I don't know, why would it fail if you don't include this?
maybe you should change your expected result from test to include the arns for Patch owned baselines if that's your issue

Reply:
This is for testing purpose i used that so i think we can remove it